packages/main/src/ComboBox.ts

```diff
@@ -327,9 +315,8 @@ class ComboBox extends UI5Element {
 	/**
 	 * Defines the filter type of the component.
 	 *
-	 * @type {sap.ui.webc.main.types.ComboBoxFilter}
-	 * @name sap.ui.webc.main.ComboBox.prototype.filter
-	 * @defaultvalue "StartsWithPerTerm"
+	 * @type {ComboBoxFilter}
+	 * @default "StartsWithPerTerm"
```

Comment:
In Storybook, there was a select previously presented, in order to choose between available options. Now, it is changed to a button with text "Set object" - is that expected?

Reply:
My mistake, it works as expected.